modin/pandas/window.py

```diff
@@ -272,3 +272,143 @@ def quantile(self, quantile, interpolation="linear", **kwargs):
                 self.axis, self.rolling_args, quantile, interpolation, **kwargs
             )
         )
+
+    def rank(
```

Comment:
why add this in this PR?

Reply:
This snuck in when I cherry-picked 3c10b00, I'll revert it.